src/format_otr/format_otr.py

```diff
@@ -1,2 +1,16 @@
-def add_one(number):
-    return number + 1
\ No newline at end of file
+from pathlib import Path
+
+
+def add_breaks_on_timestamps(text):
```

Comment:
All functions name and variables are very self explanatory. Highly appreciate it. a brief documentation following [pydocstyle](https://wiki.openpecha.org/#/articles/coding-guidelines?id=documentation) for functions would help alot for future reference.